pkg/registry/servicecatalog/broker/strategy.go

```diff
@@ -117,3 +127,29 @@ func (brokerRESTStrategy) ValidateUpdate(ctx kapi.Context, new, old runtime.Obje
 
 	return scv.ValidateBrokerUpdate(newBroker, oldBroker)
 }
+
+func (brokerStatusRESTStrategy) PrepareForUpdate(ctx kapi.Context, new, old runtime.Object) {
+	newBroker, ok := new.(*sc.Broker)
+	if !ok {
+		glog.Warning("received a non-broker object to update to")
+	}
+	oldBroker := old.(*sc.Broker)
+	if !ok {
```

Comment:
ow, yea. that's bad. I'm going to go through all the others as well. I'll bet this was originally a mistake and now it's been copied and pasted into four other places.